vignettes/analysing-n-grams.Rmd

```diff
@@ -486,3 +494,4 @@ ggplot(time_comparison, aes(pub_year, n, colour = bigrams)) +
   geom_line() +
   scale_x_continuous(breaks = scales::pretty_breaks(7))
 ```
+(#7)
```

Comment:
there should be some bottom line (figuratively, or is it literally in this case?), some EOF sentence, some closing remarks.......SOMETHING, anything.